packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxFlyoutMenu/OnyxFlyoutMenu.vue

```diff
@@ -1,9 +1,9 @@
-<script setup lang="ts" generic="TValue extends SelectOptionValue = SelectOptionValue">
+<script setup lang="ts" generic="_">
```

Comment:
```suggestion
<script setup lang="ts">
```

The generic can be removed all together, right?

Reply:
Unfortunately the component typings (build) breaks without it 😳